### DISCUSSION TITLE
Sqlalchemy Mapping Class Inheritance Hierarchies

### DISCUSSION THREAD

**Discussion by RicardoJDaleprane** [fastapi/fastapi#5849] [Questions] [answered]
I'm new to FastApi.
I am not able to add data to the database when I have generalization and specialization in database modeling.
How can I register this in the database?
I'm modeling the database using the following SqlAlchemy documentation https://docs.sqlalchemy.org/en/14/orm/inheritance.html

**Comment by anthonycorletti** [comment, marked as answer]
👋 Hey @RicardoJDaleprane! I'd recommend checking out sqlmodel's guide to inheritance 👇 

https://sqlmodel.tiangolo.com/tutorial/fastapi/multiple-models/#multiple-models-with-inheritance

Hope this helps!
